src/main/java/seedu/address/logic/parser/AddCommandParser.java

```diff
@@ -39,7 +38,7 @@ public AddCommand parse(String args) throws ParseException {
         Answer answer = ParserUtil.parseAnswer(argMultimap.getValue(PREFIX_ANSWER).get());
```

Comment:
I think we should do the isPresent check for this since `argMultimap.getValue(PREFIX_ANSWER).get()` returns an Optional.

Reply:
Same for question above

Reply:
I think I missed this out when I first edited this class.

Reply:
But does the check below no check if its present?

```
if (!arePrefixesPresent(argMultimap, PREFIX_QUESTION, PREFIX_ANSWER) || !argMultimap.getPreamble().isEmpty()) {
            throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, AddCommand.MESSAGE_USAGE));
        }
```